api/src/database/migrations/20231215A-add-focalpoints.ts

```diff
@@ -0,0 +1,13 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.alterTable('directus_files', (table) => {
+		table.json('focal_point');
```

Comment:
I'm wondering if we should normalize this to `focal_point_x` and `focal_point_y` integer columns

Reply:
My thought process was that the focal point infos are  _probably_ only needed all together. When would someone only fetch an x-coord and not the y-coord too? But not a hard requirement, easy change. Other opinions? :man_shrugging:

Reply:
It's more about database modeling and normalization than use case for the data. Storing nested JSON with a known structure isn't very sql-like, as it effectively locks away the data in a non-structured blob.